packages/zoe/src/zoeService/chargeForComputrons.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+
+import { AmountMath } from '@agoric/ertp';
+import { natSafeMath } from '../contractSupport/index.js';
+
+const { multiply, ceilDivide } = natSafeMath;
+
+const makeChargeForComputrons = (meteringConfig, feeBrand, chargeZoeFee) => {
+  const chargeForComputrons = async feePurse => {
```

Comment:
```suggestion
  /** @type {ChargeForComputrons} */
  const chargeForComputrons = async feePurse => {
```

You already declared the type. Might as well use it ;)

A type on makeChargeForComputrons would be even more helpful.

Reply:
good idea! I'll do that